docker/.env.example

```diff
@@ -38,7 +38,10 @@ LOG_LEVEL=simple
 # JWT SECRET
 ###################################################################################
 
-JWT_SECRET=randomstringthatissolongandpowerfulthatnoonecanguess
+# This JWT_SECRET is used to sign the authentication keys for user login
+# You should set it to a long randomly generated value
+# You can use this command to generate one: openssl rand -base64 128
+#JWT_SECRET=
```

Comment:
I would remove the comment so that it should as something you should populate
```suggestion
JWT_SECRET=
```

Reply:
If it's left blank, what will happen on the server, will it fail to start?

Reply:
> Will the server properly error on an empty value?

Hmm need to check that.

Reply:
![image](https://user-images.githubusercontent.com/27055614/195211680-df6dcab3-955c-481d-8c2e-67d8995c7975.png)

We are good to go with the suggestion